roles/cloud_connector/templates/proxy.conf.j2

```diff
@@ -0,0 +1,3 @@
+[Service]
+Environment=HTTPS_PROXY={{ foreman_cloud_connector_http_proxy }}
+Environment=NO_PROXY={{ foreman_cloud_connector_url }}
```

Comment:
@evgeni @ShimShtein added NO_PROXY here, this is what I interpreted by the meaning of how we want to use it here to prevent proxying between calls to Foreman

Reply:
looks correct to me